command/harness/dlite/dlite.go

```diff
@@ -137,14 +140,21 @@ func (c *dliteCommand) run(*kingpin.ParseContext) error {
 
 	g.Go(func() error {
 		<-ctx.Done()
-		return harness.Cleanup(&c.env, c.poolManager)
+		err = harness.Cleanup(&c.env, c.poolManager, true, true)
+		if c.env.Database.Postgres.Enabled {
```

Comment:
+1, do we need to call cleanup for both pool manager and distributed pool manager?
Also, this naming is confusing -> if postgres is enabled, we are using "distributed mode". Can we instead keep all the database structs the same (reuse) and then we can have a separate flag for a mode which assumes that the database is being shared across multiple instances. `<>_SHARE_DB_INSTANCE`
"Postgres" on its own doesn't indicate database sharing.